dev/launch/launch.go

```diff
@@ -1,3 +1,5 @@
+// +build darwin
```

Comment:
this file has no business being part of the linux build. it uses `launch_activate_socket` which is only provided on macOS. 

I'm unsure as to why travis picks up this file as part of `go get` but all attempts to replicate locally on linux don't run into the same problem. perhaps there's some greedy compilation configured somewhere? 

either way, this functionality is irrelevant on linux, so it seems safe to isolate this file to builds on darwin arch.

Reply:
@nateberkopec let me know if you'd prefer that i pr this fix separately.

Reply:
(for posterity; see also #225)